cognite/client/_api/data_modeling/instances.py

```diff
@@ -857,7 +857,7 @@ def list(
         instance_type: Literal["node"] = "node",
         include_typing: bool = False,
         sources: ViewIdentifier | Sequence[ViewIdentifier] | View | Sequence[View] | None = None,
-        limit: int = INSTANCES_LIST_LIMIT_DEFAULT,
+        limit: int | None = INSTANCES_LIST_LIMIT_DEFAULT,
```

Comment:
```suggestion
        limit: int | None = LIST_LIMIT_DEFAULT,
```
Suggest removing the `INSTANCES_LIST_LIMIT_DEFAULT` with same argument as above.